wysadzulice/migrations/0001_initial.py

```diff
@@ -13,31 +13,27 @@ class Migration(migrations.Migration):
         migrations.CreateModel(
             name='Campaign',
             fields=[
-                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
+                ('id', models.AutoField(primary_key=True, auto_created=True, serialize=False, verbose_name='ID')),
```

Comment:
Yep, I have found it - this attribute is set fo all fields that inherits from `Field` (root of all fields) and has default value as `True` (see: ). It is used by `serializer`s - they all inherit from `Serializer` and in its method `serialize` You have line https://github.com/django/django/blob/master/django/core/serializers/base.py#L85.

Here it is set to `False` by Django migrations - Filed `id` (or `pk` from primary key) is set internally by Django (haven't been defined in `models.py`) and so it will not be serialized (if we will intend to use default methods for serializing things in Django (which at that point we don't intent to use).